snippets/facets.liquid

```diff
@@ -22,78 +22,131 @@
   endif
 -%}
 
-<div class="facets-container">
-  {%- unless collapse_on_larger_devices -%}
+<div class="facets-container{% if filter_type == 'drawer' %} facets-container-drawer{% endif %}">
+  {%- if filter_type == 'vertical' or filter_type == 'horizontal' -%}
     <facet-filters-form class="facets small-hide">
       <form id="FacetFiltersForm" class="facets__form">
-
         {%- if results.terms -%}
           <input type="hidden" name="q" value="{{ results.terms | escape }}">
           <input name="options[prefix]" type="hidden" value="last">
         {%- endif -%}
 
         {% if enable_filtering %}
           <div id="FacetsWrapperDesktop" class="facets__wrapper">
-            {%- unless results.filters == empty -%}
-              <h2 class="facets__heading caption-large text-body">{{ 'products.facets.filter_by_label' | t }}</h2>
-            {%- endunless -%}
+            {%- if filter_type != 'vertical' -%}
+              {%- unless results.filters == empty -%}
+                <h2 class="facets__heading caption-large text-body" id="verticalTitle" tabindex="-1">{{ 'products.facets.filter_by_label' | t }}</h2>
+              {%- endunless -%}
+            {%- endif -%}
```

Comment:
```suggestion
            {%- unless filter_type == 'vertical' and results.filters == empty -%}
              <h2 class="facets__heading caption-large text-body" id="verticalTitle" tabindex="-1">{{ 'products.facets.filter_by_label' | t }}</h2>
            {%- endunless -%}
```
I think we could make this into one line instead of two. It's up to you though on whether this impacts readability.

Reply:
Good find! Grouped them but using `if`